--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -144,7 +144,6 @@
 list(APPEND APP_OPTIONS "-DLINPHONE_OUTPUT_DIR=${LINPHONE_OUTPUT_DIR}")
 list(APPEND APP_OPTIONS "-DENABLE_QT_GL=${ENABLE_VIDEO}")#Activate on video
 
-include(ExternalProject)
 set(PROJECT_BUILD_COMMAND "")
 if(CMAKE_BUILD_PARALLEL_LEVEL)
 	list(APPEND APP_OPTIONS "-DCMAKE_BUILD_PARALLEL_LEVEL=${CMAKE_BUILD_PARALLEL_LEVEL}")
@@ -181,29 +180,8 @@
 endif()
 
 
-if(NOT LINPHONE_QT_ONLY)
-ExternalProject_Add(sdk PREFIX "${CMAKE_BINARY_DIR}/sdk"
-    SOURCE_DIR "${CMAKE_SOURCE_DIR}/linphone-sdk"
-    INSTALL_DIR "${LINPHONE_OUTPUT_DIR}"
-    STAMP_DIR "${SDK_BUILD_DIR}/stamp"
-    BINARY_DIR "${SDK_BUILD_DIR}"
-    STEP_TARGETS build
-    BUILD_COMMAND ${CMAKE_COMMAND} --build <BINARY_DIR> --config $<CONFIG> ${PROJECT_BUILD_COMMAND}
-    INSTALL_COMMAND ${CMAKE_COMMAND} -E echo "Install step is already done at build time."
-    LIST_SEPARATOR | # Use the alternate list separator
-    CMAKE_ARGS ${APP_OPTIONS} ${USER_ARGS} -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR> -DCMAKE_PREFIX_PATH=${PREFIX_PATH}
-    BUILD_ALWAYS NO #${DO_BUILD}
-)
-ExternalProject_Add_Step(sdk force_build
-	COMMENT "Forcing build for 'desktop'"
-	DEPENDEES configure
-	DEPENDERS build
-	ALWAYS 1
-)
-endif()
 include(FindPkgConfig)
 
-set(APP_DEPENDS sdk)
 find_package(Qt5 5.9 COMPONENTS Core REQUIRED)
 
 if ( NOT Qt5_FOUND )
@@ -243,14 +243,3 @@
 		add_subdirectory(${CMAKE_SOURCE_DIR}/plugins "plugins-app")
 	endif()
 endif()
-ExternalProject_Add(linphone-qt-only PREFIX "${CMAKE_BINARY_DIR}/linphone-app"
-    SOURCE_DIR "${CMAKE_SOURCE_DIR}/linphone-app"
-    INSTALL_DIR "${APPLICATION_OUTPUT_DIR}"
-    BINARY_DIR "${CMAKE_BINARY_DIR}/linphone-app"
-    BUILD_COMMAND ${CMAKE_COMMAND} --build <BINARY_DIR> --config $<CONFIG> ${PROJECT_BUILD_COMMAND}
-#    INSTALL_COMMAND ${CMAKE_COMMAND} -E echo "Install step is already done at build time."
-    LIST_SEPARATOR | # Use the alternate list separator
-    CMAKE_ARGS ${APP_OPTIONS} ${USER_ARGS} -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR> -DCMAKE_PREFIX_PATH=${PREFIX_PATH}
-    EXCLUDE_FROM_ALL ON
-    #BUILD_ALWAYS ON
-)

