--- a/linphone-app/CMakeLists.txt
+++ b/linphone-app/CMakeLists.txt
@@ -22,8 +22,8 @@
 cmake_minimum_required(VERSION 3.1)
 
 find_package(bctoolbox CONFIG)
-set(FULL_VERSION )
-bc_compute_full_version(FULL_VERSION)
+set(FULL_VERSION "@FULLVERSION@")
+#bc_compute_full_version(FULL_VERSION)
 set(version_major )
 set(version_minor )
 set(version_patch )
@@ -137,7 +137,12 @@
 	endif ()
 endforeach ()
 
-bc_git_version(${TARGET_NAME} ${PROJECT_VERSION})
+add_custom_target(linphone-git-version
+			WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+			COMMAND echo "#define MS2_GIT_VERSION \"0000000\"" > "gitversion.h"
+			BYPRODUCTS "gitversion.h"
+			VERBATIM
+		)
 
 #-------------------------------------------------
 set(ASSETS_DIR "assets")
--- a/linphone-app/cmake_builder/linphone_package/CMakeLists.txt
+++ b/linphone-app/cmake_builder/linphone_package/CMakeLists.txt
@@ -27,7 +27,7 @@
 endif()
 include("${CMAKE_CURRENT_SOURCE_DIR}/../../application_info.cmake")
 
-find_package(Git)
+#find_package(Git)
 #Policy set to allow link from other directory (cmake 3.13)
 if (POLICY CMP0079)
   cmake_policy(SET CMP0079 NEW)
@@ -39,7 +39,8 @@
 # ==============================================================================
 # Build package version.
 # ==============================================================================
-bc_compute_full_version(APP_PROJECT_VERSION)
+#bc_compute_full_version(APP_PROJECT_VERSION)
+set(APP_PROJECT_VERSION "@FULLVERSION@")
 if (GIT_EXECUTABLE AND NOT(APP_PROJECT_VERSION))
   execute_process(
     COMMAND ${GIT_EXECUTABLE} describe --always
--- a/linphone-app/build/CMakeLists.txt
+++ b/linphone-app/build/CMakeLists.txt
@@ -46,7 +46,8 @@
   "libmng.spec"
 )
 
-bc_compute_full_version(PROJECT_VERSION_BUILD)
+set(PROJECT_VERSION_BUILD "@FULLVERSION@")
+#bc_compute_full_version(PROJECT_VERSION_BUILD)
 if(PROJECT_VERSION_BUILD)
   set(CPACK_PACKAGE_FILE_NAME "${CPACK_PACKAGE_NAME}-${PROJECT_VERSION_BUILD}")
 endif()

