--- a/linphone-app/CMakeLists.txt
+++ b/linphone-app/CMakeLists.txt
@@ -22,8 +22,8 @@
 cmake_minimum_required(VERSION 3.1)
 
 find_package(bctoolbox CONFIG)
-set(FULL_VERSION )
-bc_compute_full_version(FULL_VERSION)
+set(FULL_VERSION "@FULLVERSION@")
+#bc_compute_full_version(FULL_VERSION)
 set(version_major )
 set(version_minor )
 set(version_patch )
--- a/linphone-app/cmake_builder/linphone_package/CMakeLists.txt
+++ b/linphone-app/cmake_builder/linphone_package/CMakeLists.txt
@@ -27,7 +27,7 @@
 endif()
 include("${CMAKE_CURRENT_SOURCE_DIR}/../../application_info.cmake")
 
-find_package(Git)
+#find_package(Git)
 #Policy set to allow link from other directory (cmake 3.13)
 if (POLICY CMP0079)
   cmake_policy(SET CMP0079 NEW)
@@ -39,7 +39,8 @@
 # ==============================================================================
 # Build package version.
 # ==============================================================================
-bc_compute_full_version(APP_PROJECT_VERSION)
+#bc_compute_full_version(APP_PROJECT_VERSION)
+set(APP_PROJECT_VERSION "@FULLVERSION@")
 if (GIT_EXECUTABLE AND NOT(APP_PROJECT_VERSION))
   execute_process(
     COMMAND ${GIT_EXECUTABLE} describe --always

